llzk_backend/src/codegen.rs

```diff
@@ -1,4 +1,6 @@
+#![allow(unused)] // TODO: TEMP
```

Comment:
If too many warnings are making it hard to find build errors this helps: https://github.com/cargo-limit/cargo-limit